arch/arm/soc/st_stm32/stm32f7/soc.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2018 Yurii Hamann
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file SoC configuration macros for the ST STM32F7 family processors.
+ *
+ * Based on reference manual:
+ * RM0385 Reference manual STM32F75xxx and STM32F74xxx
+ *   advanced ARM(r)-based 32-bit MCUs
+ *
+ * Chapter 2.2.2: Memory map and register boundary addresses
+ */
+
+#ifndef _STM32F7_SOC_H_
+#define _STM32F7_SOC_H_
+
+#ifndef _ASMLANGUAGE
+
+#include <device.h>
+#include <misc/util.h>
+#include <stm32f7xx.h>
+
+#include "soc_irq.h"
+
+#endif /* !_ASMLANGUAGE */
+
+#endif /* _STM3274_SOC_H_ */
```

Comment:
Given you need to respin the patchset, a small typo here: _STM3274_SOC_H_ => _STM32F7_SOC_H_

Reply:
Thank you. I will fix it.